src/models/emergency.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+/** 
+  An Emergency can be created by users and confirmed by admins
+ */
+
+const EmergencySchema = new Schema({
+  bloodGroup: {
+    type: String,
+    enum: ["A+", "A-", "B+", "B-", "AB+", "AB-", "O+", "O-"],
+    required: true,
```

Comment:
We'll use schema validators provided by mongoose, this should be added to other schemas too, but let's start with this:
```suggestion
    required: [true, 'Blood group is required']
```